index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+    <link rel="stylesheet" type="text/css" href="css/splash.css">
+    <link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet">
+</head>
+<body>
+    <div class="container">
+        <div class="splash">
+            <img src="assets/gameTitle.png" alt="Mayrio Kart">
+            <button id="pullInstructions"  class="instructionsBtn">
+                Instructions
+            </button>
+            <div class=" gameLevel">
+            <div class="instructions" id="instructions"></div>
```

Comment:
having a class and an id on the same element with the same name isnt very verbose

Reply:
Removed id attribute and changed instructions.js file to point to the class rather than the id